components/finance/FinanceCategoriesList/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import FinanceCategoriesListWrapper from '../FinanceCategoriesListWrapper'
+
+function FinanceCategoriesList({ item }: any) {
+  return (
+    <FinanceCategoriesListWrapper>
+      <li key={item.name} className="flex mt-px">
+        <div className="p-2.5 flex-1 bg-workItem">{item.name}</div>
+        <div
+          className={`p-2.5 w-24 text-center ${
+            item.incomeOrExpense === 'expense' ? 'bg-plannedExpense' : 'bg-plannedIncome'
+          }`}
+        >
+          {item.totalAmount.reduce((a: any, b: any) => ({ amount: a.amount + b.amount })).amount}
+        </div>
```

Comment:
this entire content inside <li> should be part of <FinanceCategoriesListItem/>

Reply:
fixed.